lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -131,16 +131,17 @@ export default function LaunchErrorDialog({
         </ErrorModal>
       );
 
+    case 'blackboard_group_set_not_found':
```

Comment:
I think making this generic works now, like we do for `blackboard|canvas_group_set_empty`.

It can be split later if LMS specific language is needed.